### PR TITLE
Add Schedule format

### PR DESCRIPTION
**Pull request by marciw-da**
Backporting new Schedule format section from 3.0.0 -- see also #556

### CODE REVIEW

**Review by garyverhaegen-da — APPROVED** [review]

File: docs/2.9.0/docs/canton/usermanual/pruning.rst [hunk 1]
```diff
@@ -40,9 +44,81 @@ Refer to the cron specification to customize the pruning schedule. Here are a fe
...
+   * - day of the week
+     - number from ``1`` to ``7`` (``1`` = Sunday) 
+       |br|
+       **or** 
+       |br|
+       the first three letters of the day name (``SUN``, ``MON``, ``TUE``, ...)
+   * - year (optional)
+     - number from ``1900`` to ``2099``
+
+*\*Ranges specified with "to" are inclusive of both endpoints.*
```
Comment (garyverhaegen-da) [inline docs/2.9.0/docs/canton/usermanual/pruning.rst:84]: This sentence threw me off - it took me a while to figure out that it refers to the ranges specified in the table above, and not to ranges the user would define.

How bad would it be layout-wise to just say "included" in all relevant entries in the table directly? (And then remove this sentence.) Alternatively,

```suggestion
*\*Ranges specified with "to" in the table above are inclusive of both endpoints.*
```

Comment (marciw-da) [reply docs/2.9.0/docs/canton/usermanual/pruning.rst:84]: Hmm. Were you looking at just the .rst or also the rendered view? There's an asterisk that connects the column heading to the note (you can also see it here https://docs.daml.com/3.0.0/canton/usermanual/pruning.html#schedule-format)

Maybe I'll go with "In the table, ranges specified..." 

table

Comment (marciw-da) [reply docs/2.9.0/docs/canton/usermanual/pruning.rst:84]: I can also look at putting the note right there under the column heading, though I think I tried that and it was messy. But I'll check.

Comment (garyverhaegen-da) [reply docs/2.9.0/docs/canton/usermanual/pruning.rst:84]: Ah, sorry, no, I was just reading through the rst. It is indeed a lot clearer with the layout.

**Pull request merged**